.cicd/build.sh

```diff
@@ -19,7 +19,7 @@ else # Linux
 
     # PRE_COMMANDS: Executed pre-cmake
     PRE_COMMANDS="cd $MOUNTED_DIR/build"
-    BUILD_COMMANDS="cmake .. && make -j$JOBS"
+    BUILD_COMMANDS="cmake .. -DEOSIO_RUN_INTEGRATION_TESTS=True -DEOSIO_ROOT=/usr/local && make -j$JOBS"
```

Comment:
Standard cmake bool variables use 'ON/OFF' for their values as documented in https://cmake.org/cmake/help/latest/command/set.html. I would prefer to adopt that convention. i.e. `-DEOSIO_RUN_INTEGRATION_TESTS=ON`.

Reply:
Good catch. Fixed.